substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java

```diff
@@ -77,9 +79,38 @@ public static boolean parseOnce() {
     @Option(help = "Name of the main entry point method. Optional if --shared is used.")//
     public static final HostedOptionKey<String> Method = new HostedOptionKey<>("main");
 
+    @Option(help = "Directory of the image file to be generated", type = OptionType.User)//
+    public static final HostedOptionKey<String> Path = new HostedOptionKey<>(null);
+
     @Option(help = "Name of the output file to be generated", type = OptionType.User)//
```

Comment:
Since Name is now more than that, can you please update the help message to reflect that?
Thanks